packages/fastify/src/withClerkMiddleware.ts

```diff
@@ -63,3 +67,45 @@ export const withClerkMiddleware = (options: ClerkFastifyOptions) => {
     req.auth = requestState.toAuth();
   };
 };
+
+const getAbsoluteUrlFromHeaders = (url: string, headers: Headers): URL => {
```

Comment:
🔧 i believe this code should not be in `fastify` package but in `backend`. Why is it here?

Reply:
It should be here, because every package is responsible to create the `Request` object itself.